openwisp_radius/settings.py

```diff
@@ -26,6 +26,7 @@ def get_settings_value(option, default):
 
 
 RADIUS_API = get_settings_value('API', True)
+RADIUS_API_BASEURL = get_settings_value('API_BASEURL', '/radiusbatch/csv/')
```

Comment:
I don't understand, the default API BASE URL is `/radiusbatch/csv/'`?
It's really confusing.